app/assets/javascripts/analytics/pii.js

```diff
@@ -75,6 +92,18 @@
     return array
   }
 
+  pii.prototype.stripQueryStringParameters = function (string) {
+    for (var i = 0; i < this.queryStringParametersToStrip.length; i++) {
+      var parameter = this.queryStringParametersToStrip[i]
+      var escaped = parameter.replace(ESCAPE_REGEX_PATTERN, '\\$&')
+      var regexp = new RegExp('((?:\\?|&)' + escaped + '=)(?:[^&#\\s]*)', 'g')
+      var replacement = '[' + parameter + ']'
```

Comment:
Doesn't look like this variable gets used anywhere?

Reply:
Oops - thanks for catching.